### PR TITLE
Apple old site from internet archieve

### PR DESCRIPTION
**Pull request by dipbazz**
## Definition

We have used several techniques like flexbox, position, float, and gradient to layout our and style our website. And it's about apple's iPhone 5 from apple's old website in archives.

## About the Project

This project is about building apple's old website from the archives while making use of flex, positioning, and gradients.

The features of this project are listed below:
1. Website has a header containing:
    - Navbar With  apple logo, menu links, input search
      - Navbar has a linear gradient
    - Background image
    - banner links
2. Has a gallery
    - Images of apple products
    - The gallery items are links
3. Footer section with two main parts
    - The top footer
      - The top footer contains sales links and useful site links
    - The bottom footer
      - The bottom footer contains terms, policy, and rights

### CODE REVIEW

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Hello Team!! Great work on the apple clone!!

:100: Great README!
:100: Descriptive Pull Request message.
:100: Descriptive enough commit messages.
:100: Updated license with Author and Year.
:100: Great looking website.

### [Mandatory] :red_circle: 

- [x] Use more appropriate HTML tags for navbar's structure. While ```<span>``` works, is not the correct approach when creating a navbar. Please use an unordered list as the project does with the footer menu. See comments.

- [x] All image tags should have an alternate text. Please don't leave it blank.

- [x] Links in nav are using a unnecessary ```<span>``` wrapper. Please remove it. See inline comment

### [Comments] :green_circle: 

- The <span> tag is an inline container used to mark up a part of a text, or a part of a document. Example.
```<p>My mother has <span style="color:blue">blue</span> eyes.</p>```

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    <nav>
+      <span class="nav-items">
+        <a href="#" class="nav-links">
+          <span class="iconify apple-logo" data-icon="ant-design:apple-filled" data-inline="false"></span>
+        </a>
+      </span>
+
+      <span class="nav-items">
+        <a href="#" class="nav-links">
+          <span>Store</span>
```
Comment (jubaan) [inline index.html]: - [x] This ```<span>``` wrapper is unnecessary. This applies to the other elements that match this pattern in the navbar. Except for the apple, and search icon that uses iconify syntax.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+          </span>
+        </label>
+        <input type="search" name="search" id="search">
+      </form>
+
+    </nav>
+    <div class="banner-text">
+      <figure class="text-img-top">
+        <a href="#">
+          <img src="images/what.png" alt="" >
```
Comment (jubaan) [inline index.html]: - [x] Images are missing alternative text. Please fill it with the appropriate information.

**Review by jubaan — CHANGES_REQUESTED** [review]

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+          </label>
+          <input type="search" name="search" id="search">
+        </li>
+
+      </ul>
+    </nav>
+    <div class="banner-text">
+      <figure class="text-img-top">
+        <a href="#">
+          <img src="images/what.png" alt="what will your verese be" >
```
Comment (umairarshadbutt) [inline index.html]: Text is missing, Replace image with text and manage it

File: index.html [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+        </a>
+      </figure>
+
+      <figure class="text-img-mid">
+        <a href="#">
+          <img src="images/everyone.png" alt="everyone has something to share, see how people are using ipad in thier unique ways to add to the world story" >
```
Comment (umairarshadbutt) [inline index.html]: Text is missing, Replace image with text and manage it

File: index.html [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+        </a>
+      </figure>
+
+      <figure class="text-img-bot">
+        <a href="#" class="img-link">
+          <img src="images/watch.png" alt="watch the your verse film and explore the stories" >
```
Comment (umairarshadbutt) [inline index.html]: Text is missing, Replace image with text and manage it

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @Misori-simon and @dipbazz 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly work on the suggested changes 👇.
- [x] Add Mac in navbar
- [x] Text is missing, Replace image with text and manage it
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
### The [clone](https://dipbazz.github.io/building-with-backgrounds-and-gradients/) shows a good understanding of Backgrounds and Gradients. :100:
##
- The cloned page is rendering as it should and meets all the project specifications.
- The project has a descriptive README.md and is professionally done. Good job with that.
- CSS is clean. No linter errors at this point and no bad practices used as well.
- The markup as well has no errors nor warnings to show. Good job with that.
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @Misori-simon and @dipbazz! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
**If you have any questions, let me know on slack at any time. I will be happy to help.**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Pull request merged**